### DISCUSSION TITLE
What to do if briefcase new doesn't work?

### DISCUSSION THREAD

**Discussion by Slonit** [beeware/briefcase#1651] [Q&A] [answered]
I decided to make an application using BeeWare Tutorial, here is the link: https://docs.beeware.org/en/latest/. In Lesson 0, I downloaded Git, but didn't download Winget - it doesn't work for me. Then, when I had done everything, in Lesson 1 I reached the point where I need to insert briefcase new b into the terminal, and at this point everything stops working. Below in the screenshot it says that you need to download Git, but I downloaded it and pasted it into the same folder where the file is. I also rebooted the terminal using the Reload button, but it doesn't work again. Now I don’t understand anything what to do and I haven’t found this problem anywhere on the Internet. Help me please!
![Безымянный](https://github.com/beeware/briefcase/assets/143531646/57291e18-97c5-4691-b103-d73dbce9d5c3)

**Comment by rmartin16** [comment, marked as answer]
When you download git from https://git-scm.com, you will need to run the downloaded file so that git is installed on your machine. You will know it is installed correctly when running the command `git` shows the list of git commands.

**Reply by Slonit** [reply]
Thank you very much, you helped a lot, I didn’t know. But I don’t know what to do next, can you explain?
![Безымянный1](https://github.com/beeware/briefcase/assets/143531646/a4ec2c49-08d5-442e-a06d-a5adc83044f9)

**Reply by rmartin16** [reply]
It looks like git is installed on your machine now. So, I would restart your computer to make sure everything is in place and then continue on in the BeeWare tutorial.

**Reply by Slonit** [reply]
Thank you very much, everything worked out, otherwise I’ve been suffering for two weeks!
